core/src/main/java/de/danoeh/antennapod/core/storage/FeedCursorMapper.java

```diff
@@ -0,0 +1,70 @@
+package de.danoeh.antennapod.core.storage;
```

Comment:
Oh, sorry for the double-post. I just noticed that we already have a `de.danoeh.antennapod.core.storage.mapper` package. It would fit there even better :)

Reply:
Done.